chemicaltoolbox/im-pipelines/featurestein-generate.xml

```diff
@@ -0,0 +1,67 @@
+<tool id="featurestein_generate" name="Generate 'FeatureStein' merged features maps" version="0.1">
```

Comment:
suggestion: use the same version number as the im-pipelines repo, i.e. 1.1.3

Reply:
oh I see you did already in the macro - then just use `@TOOL_VERSION@+@GALAXY_VERSION@`